modules/kafka/kafka.go

```diff
@@ -30,6 +31,8 @@ echo '' > /etc/confluent/docker/ensure
 	// }
 )
 
+var timeToWaitUntilContainerMapPorts = time.Second
```

Comment:
I'm not sure about the one second time sleep.

I've been doing some testing because I had the original problem as in the issue for which this PR was created for, and so I tried to implement a workaround in my project as follows:

```
	sleepCommand := testcontainers.NewRawCommand([]string{"sleep", "1"})

	kafkaContainer, err := tcKafka.RunContainer(
		ctx,
		tcKafka.WithClusterID("test-kafka-cluster"),
		testcontainers.WithImage("confluentinc/cp-kafka:latest"),
		testcontainers.WithStartupCommand(sleepCommand),
	)
```

My assumption with this code was that the `WithStartupCommand` command would end up running before the PostStartLifecycleHook, so we would end up waiting for a second right after the container is started.

But when I ran the test multiple times, I would eventually end up running into the same `port not found` error.

I had a bit more luck with sleeping for 1.5 seconds, but this is just anecdotal. I don't know what the optimal sleep time is to ensure that the kafka container has started enough to be able to get its mappedport.

Reply:
Actually, nevermind. I looked into the code for the `WithStartupCommand` function and found out that the sleep command would be executed at the end of all the default configured `PostStartLifecycleHook`, so my assumption was incorrect